src/date.rs

```diff
@@ -492,11 +492,18 @@ mod test {
                 .to_string(),
             "2018-02-13T23:08:32.123000000Z"
         );
+        #[cfg(not(target_os = "windows"))]
         assert_eq!(
             format_rfc3339_nanos(UNIX_EPOCH + Duration::new(1_518_563_312, 789_456_123))
                 .to_string(),
             "2018-02-13T23:08:32.789456123Z"
         );
+        #[cfg(target_os = "windows")] // Not sure what is up with Windows rounding?
```

Comment:
@ChrisDenton do you happen to know why this might be? Is it expected? Could I do something different to avoid this?

Reply:
See the docs for the underlying system data structure: https://learn.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-filetime

> Contains a 64-bit value representing the number of 100-nanosecond intervals since January 1, 1601 (UTC).

There's no easy fix as it's inherent in how `SystemTime` is defined. I mean, other than manually calculating the nano second difference and adjusting the result accordingly.